src/openzeppelin/access/ownable.cairo

```diff
@@ -47,6 +55,81 @@ func Ownable_transfer_ownership{
         range_check_ptr
     }(new_owner: felt) -> (new_owner: felt):
     Ownable_only_owner()
+    with_attr error_message("Ownable: owner cannot be zero address"):
+        assert_not_zero(new_owner)
+    end 
     Ownable_owner.write(new_owner)
     return (new_owner=new_owner)
 end
+
+# Propose a new ownner which will need to accept before the ownership is transferred.
+func Ownable_propose_owner{
+        syscall_ptr : felt*, 
+        pedersen_ptr : HashBuiltin*,
+        range_check_ptr
+    }(new_owner: felt):
+    Ownable_only_owner()
+    with_attr error_message("Ownable: owner cannot be zero address"):
+        assert_not_zero(new_owner)
```

Comment:
> Het @ctrlc03 thank you for submitting this PR. This looks really cool! We've been deliberating on using namespace in the contracts and reconsidering new conventions for function naming (see the discussion in #267). Unfortunately, reviewing PRs has been blocked because of this, but we'll take a look at this as soon as we can!

Will review that discussion. Should I be amending the whole library to use namespaces instead?

Reply:
Hmm so here's my thoughts: #244 already has the namespace updates for Ownable, and it might be wise to wait for that and #240 to be resolved/merged. I don't want to recommend changes that are still being ironed out. As long as the logic is sound and tested, refactoring with the new convention should be relatively quick and simple.

Reply:
Fair enough I will wait for that and in the meantime figure out how to write the tests with pytest. I run into plenty of errors when trying to run the ones already defined in the repo, must be due to some issue with my local setup. I did test the code after deploying it but I do agree that pytest should be there too, might take me some time to implement depending on my schedule. 

Thanks for having a look at this PR.

Reply:
My pleasure, thanks for submitting the PR! 

> I run into plenty of errors when trying to run the ones already defined in the repo, must be due to some issue with my local setup.

Yeah, definitely check that all dependencies are up to date and that you're running tox (not pytest directly) from the `cairo-contracts` directory.